angular-spa-ui/js/search-app/tags-module/templates/tag-item.html

```diff
@@ -0,0 +1,3 @@
+<script id="tagItemTemplate.html" type="text/ng-template">
+    <span class="tag" tid="{{tag._id}}" style="color: {{tag.textColor}}; background-color:{{tag.backgroundColor}};" >{{tag.name}}</span>
```

Comment:
Comment for Friday:
Variable in '{{' '}}' brackets should be separated with space:
{{ tag.backgroundColor }}